### PR TITLE
Blog routes

### PR DESCRIPTION
**Pull request by macitege**

### CODE REVIEW

**Comment by MaxxKowalik** [comment]
Lookin fly

File: app/models/blog.js [hunk 1]
```diff
@@ -9,10 +9,9 @@ const blogSchema = new mongoose.Schema({
     type: String,
     required: true
   },
-  author: {
+  owner: {
     type: mongoose.Schema.Types.ObjectId,
-    ref: 'User',
-    required: true
```
Comment (discocisco) [inline app/models/blog.js]: Will work fine regardless, but we might in the future want to add back the `required: true` for blogs.owner

**Comment by discocisco** [comment]
Reviewed and can merge.

**Comment by rogerdunnhawaii** [comment]
good by me

**Pull request merged**